contracts/modules/GoldenEggMetadata.sol

```diff
@@ -2,38 +2,83 @@
 pragma solidity ^0.8.16;
 
 import { LibString } from "solady/utils/LibString.sol";
+import { OwnableRoles } from "solady/auth/OwnableRoles.sol";
 import { ISoundEditionV1 } from "@core/interfaces/ISoundEditionV1.sol";
 import { IGoldenEggMetadata } from "@modules/interfaces/IGoldenEggMetadata.sol";
 
 contract GoldenEggMetadata is IGoldenEggMetadata {
+    // =============================================================
+    //                            STORAGE
+    // =============================================================
+
+    /**
+     * @dev The maximum `tokenId` for `edition` that has a numbered json.
+     * If zero, all `tokenId`s have number jsons.
+     */
+    mapping(address => uint256) public numberedUpto;
+
+    // =============================================================
+    //               PUBLIC / EXTERNAL WRITE FUNCTIONS
+    // =============================================================
+
+    /**
+     * @inheritdoc IGoldenEggMetadata
+     */
+    function setNumberedUpto(address edition, uint256 tokenId) external onlyEditionOwnerOrAdmin(edition) {
```

Comment:
```suggestion
    function setNumberedUpTo(address edition, uint256 tokenId) external onlyEditionOwnerOrAdmin(edition) {
```

How should this be set? Should we have some default e.g. 1000 so that we can avoid making this tx unless skipping that default?

Reply:
Ok, renamed. "up to" should be two words.